### PR TITLE
fixed: Exposure of audio and video bitrate for GUIInfoManager

### PR DESCRIPTION
**Pull request by arnova**
This is partially a proxy PR for FernetMenta fixing the audio bitrate info exposure in VideoPlayer. Additionally I fixed the video bitrate info in VideoPlayer and made them available from our GUIInfoManager so it can be used with skins again (so we can have this info again like with Jarvis).

## Types of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by arnova** [comment]
@FernetMenta : Please check the other bits as well. Note: Turns out we can't use musicplayer.-labels while playing video so I created a separate label for that.

File: xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp [hunk 1]
```diff
@@ -1397,6 +1397,7 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
         st->fAspect = SelectAspect(pStream, st->bForcedAspect) * pStream->codec->width / pStream->codec->height;
         st->iOrientation = 0;
         st->iBitsPerPixel = pStream->codec->bits_per_coded_sample;
+        st->iBitRate = pStream->codec->bit_rate;
```
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp]: codec is deprecated, use codecpar

Comment (arnova) [inline xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp]: Ok, but does this also apply to the line above?

**Comment by arnova** [comment]
@MartijnKaijser / @FernetMenta : Do we want this for Krypton? And if so does it make sense to update our skins to show video/audio bitrate info in the video (process) info screen?

File: xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp (hunk 1 above)
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp]: only to this change. existing code is out of scope here.

**Comment by arnova** [comment]
@FernetMenta : Updated with codecpar

**Comment by FernetMenta** [comment]
jenkins build this please

**Comment by FernetMenta** [comment]
Android build error unrelated

**Comment by arnova** [comment]
ping @MartijnKaijser : backport?

**Comment by MartijnKaijser** [comment]
I don't think it's really warranted as backport cause it's a missing feature. It also requires skin changes as it exposes new labels. Ping @ronie

**Comment by ronie** [comment]
think it should be crash-fixes only by now?

**Comment by gade01** [comment]
What are the infolabels for this PR?

I tried VideoPlayer.VideoBitrate and VideoPlayer.AudioBitrate but nothing displays.

**Comment by arnova** [comment]
@gade01 : Yes, those should work. Are you sure you've used a sample that also reports kb/s info MediaInfo  (there are samples out there that don't have it).

**Comment by gade01** [comment]
> Are you sure you've used a sample that also reports kb/s info MediaInfo (there are samples out there that don't have it).

@arnova: Yeah, that must have been it. I tried several files before, but now it displays fine with another file.

Thanks for the PR :)

**Pull request merged**